packages/lodestar/src/network/peers/peerManager.ts

```diff
@@ -45,6 +46,11 @@ export type PeerManagerOpts = {
   targetPeers: number;
   /** The maximum number of peers we allow (exceptions for subnet peers) */
   maxPeers: number;
+  /**
+   * Delay the 1st heartbeat after starting discv5
+   * See https://github.com/ChainSafe/lodestar/issues/3423
+   */
+  firstHeartBeatDelayMs: number;
```

Comment:
Do we want to delay the whole heartbeat or only discv5 queries?

Reply:
Another approach is to record the start time of discv5 and in this block:

https://github.com/ChainSafe/lodestar/blob/c5cef290fd3eab076fc9872bce05d10f913e7dc0/packages/lodestar/src/network/peers/peerManager.ts#L403-L405

Check

```ts
if (Date.now() - this.discv5StartMs > this.discv5FirstQueryDelayMs) {
  // do query
}
```

Reply:
discv5 queries is part of the `heartbeat` so I delayed the whole heartbeat only at the 1st time to make it simple. The default delay in ms is 1000 (1s)